draft-ietf-jsonpath-base.md

```diff
@@ -1398,17 +1398,17 @@ Queries:
 
 A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node. For example,
 the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
-`$['book'][3]` and `$['book'][5]`. For a given JSON document, there is a one to one correspondence between the document's
+`$['book'][3]` and `$['book'][5]`. For a given JSON value, there is a one to one correspondence between the value's
 nodes and the Normalized Paths that identify these nodes.
 
 A JSONPath implementation may output Normalized Paths instead of, or in addition to, the values identified by these paths.
 
 Since bracket notation is more general than dot notation, it is used to construct Normalized Paths.
 Single quotes are used to delimit string member names. This reduces the number of characters that
-need escaping when Normalized Paths appear as strings (which are delimited with double quotes) in JSON documents.
+need escaping when Normalized Paths appear as strings (which are delimited with double quotes) in JSON values.
```

Comment:
I think for once we actually mean JSON texts here...

Reply:
Fixed.